extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/videos/GoogleVideosExporter.java

```diff
@@ -47,17 +48,20 @@ public class GoogleVideosExporter
   private final GoogleCredentialFactory credentialFactory;
   private volatile GoogleVideosInterface videosInterface;
   private JsonFactory jsonFactory;
+  private final Monitor monitor;
 
-  public GoogleVideosExporter(GoogleCredentialFactory credentialFactory, JsonFactory jsonFactory) {
+  public GoogleVideosExporter(GoogleCredentialFactory credentialFactory, JsonFactory jsonFactory, Monitor monitor) {
     this.credentialFactory = credentialFactory;
     this.jsonFactory = jsonFactory;
+    this.monitor = monitor;
   }
 
   @VisibleForTesting
   GoogleVideosExporter(
-      GoogleCredentialFactory credentialFactory, GoogleVideosInterface videosInterface) {
+      GoogleCredentialFactory credentialFactory, GoogleVideosInterface videosInterface, Monitor monitor) {
     this.credentialFactory = credentialFactory;
     this.videosInterface = videosInterface;
+    this.monitor = monitor;
   }
```

Comment:
In the future, we should be sure to do these types of changes in separate PRs. I think its ok for this PR

Reply:
Makes sense